src/_pytest/terminal.py

```diff
@@ -1409,7 +1409,10 @@ def _get_line_with_reprcrash_message(
         pass
     else:
         if not running_on_ci():
```

Comment:
Instead of the code you wrote below, let us rewrite this section to consider -vv and on CI to the same case:

```python
        # Do not truncate summary information on CI or if -vv is passed.
        if running_on_ci() or config.option.verbose >= 2:
            msg = f" - {msg}"
        else:
            available_width = tw.fullwidth - line_width
            msg = _format_trimmed(" - {}", msg, available_width)
```

Reply:
Done!